lib/util.js

```diff
@@ -517,7 +517,7 @@ function formatCollectionIterator(ctx, value, recurseTimes, visibleKeys, keys) {
   var nextRecurseTimes = recurseTimes === null ? null : recurseTimes - 1;
   var vals = mirror.preview();
   var output = [];
-  for (let o of vals) {
+  for (const o of vals) {
```

Comment:
Hmm. Is the REPL possibly evaluated in strong mode?

This prints 1,1,1:
````js
"use strong"
for (const a of [1,2,3]) console.log(a)
```

This prints 1,2,3:
````js
"use strict"
for (const a of [1,2,3]) console.log(a)
```